### PR TITLE
Ann docs edits

### PR DESCRIPTION
**Pull request by erinys**
- Cleaned up front page
image

- Added new repo as a separate section

- Removed mentions to write operations

- Added Hoyt's API docs

### CODE REVIEW

File: docs/markdowns/filesystem.md [hunk 1]
```diff
@@ -1,37 +1,82 @@
...
+
+  # Open a file from the repository.
+  f = fs.open('results.csv')
+
+  # Read the contents of the file.
+  contents = f.read()
+```
+
+## fsspec integration
+
```
Comment (rajatarya) [inline docs/markdowns/filesystem.md:64]: Let's add the fsspec link here:

[Filesystem (fsspec)](https://filesystem-spec.readthedocs.io/en/latest/usage.html)

I see other packages doing this so people interested in the spec can follow it.

File: docs/markdowns/filesystem.md [hunk 2]
```diff
@@ -1,37 +1,82 @@
...
+Many packages such as pandas and pyarrow support the fsspec protocol.
+xet:// URLs must be used as file paths in these packages. For example, to read a csv from pandas, use:
+
+```sh
+  import pyxet
```
Comment (rajatarya) [inline docs/markdowns/filesystem.md:69]: This needs `import pandas as pd` or else it won't work. Pandas needs to imported for the line to work.

File: docs/markdowns/quickstart.md [hunk 3]
```diff
@@ -54,57 +104,5 @@ info['features'] = features
...
-    * Sharing data
-    * Pushing code
-    * Running experiments
-    * Saving models
-    * Saving logs
-
-Checkout this [titanic app](https://xethub.com/xdssio/titanic-server-example), for a more comprehensive example.
-
-
+Continue to see how you can use pyxet on a private XetHub repository.
```
Comment (rajatarya) [inline docs/markdowns/quickstart.md:107]: Continue to where?

**Review by rajatarya — COMMENTED** [review]
Looking good, a couple minor comments!

File: docs/markdowns/quickstart.md (hunk 3 above)
Comment (erinys) [reply docs/markdowns/quickstart.md:107]: image

I'll add "to the next section" to help clarify.

**Review by rajatarya — APPROVED** [review]
Looks good! Will publish once latest package is published to PyPI.

**Pull request merged**